labelbox/client.py

```diff
@@ -636,16 +637,33 @@ def create_project(self, **kwargs) -> Project:
             description (str): A short summary for the project
             media_type (MediaType): The type of assets that this project will accept
             queue_mode (Optional[QueueMode]): The queue mode to use
-            auto_audit_percentage (Optional[float]): The percentage of data rows that will require more than 1 label
-            auto_audit_number_of_labels (Optional[float]): Number of labels required for data rows selected for multiple labeling (auto_audit_percentage)
+            quality_mode (Optional[QualityMode]): The quality mode to use (e.g. Benchmark, Consensus). Defaults to
+                Benchmark
         Returns:
             A new Project object.
         Raises:
             InvalidAttributeError: If the Project type does not contain
                 any of the attribute names given in kwargs.
         """
-        media_type = kwargs.get("media_type")
+
+        auto_audit_percentage = kwargs.get("auto_audit_percentage")
```

Comment:
if these attributes are removed from the model, why are we dealing with them here?

Reply:
This is here to help present a more readable message that can inform users how to migrate to the new usage model